AWSome.Bucket/Config/Preferences.cs

```diff
@@ -2,32 +2,38 @@
 using Amazon.Runtime;
 using Amazon.Runtime.CredentialManagement;
 using Amazon.S3;
-using System.Collections.Specialized;
-using System.Configuration;
 
-namespace AWSome.Bucket.Config
+namespace Liberator.AWSome.Bucket.Config
 {
+    /// <summary>
+    /// 
+    /// </summary>
     public class Preferences
     {
-        public static NameValueCollection appSettings = ConfigurationManager.AppSettings;
+        /// <summary>
+        /// 
+        /// </summary>
         public static AWSCredentials UserAWSCredentials { get; set; }
+
+        /// <summary>
+        /// 
+        /// </summary>
         public static RegionEndpoint RegionEndpoint { get; set; }
 
+        /// <summary>
+        /// 
+        /// </summary>
         public static string ProfileName { get; set; }
-        public static string ProfileLocation { get; set; }
 
         /// <summary>
-        /// Sets the Preferences based on the configuration file
+        /// 
         /// </summary>
-        public Preferences()
-        {
-        }
-
-
-        // <summary>
-        // Gets the DynamoDB Client set in the App.config file
-        // </summary>
-        // <returns>The Dynamo DB Client as configured</returns>
+        public static string ProfileLocation { get; set; }
+        
+        /// <summary>
+        /// Gets the Amazon S3 Client set in the App.config file
+        /// </summary>
+        /// <returns>The Amazon S3 Client as configured</returns>
         public static IAmazonS3 GetAmazonS3Client()
         {
             UserAWSCredentials = GetAWSCredentials();
```

Comment:
Lots of empty summaries in this class? On purpose?

Reply:
Yes …

… nope, can't keep up the lie. I thought I had filled them all in. LOL